aws/service.go

```diff
@@ -1529,6 +1537,11 @@ func getDefaultAwsRegion(d *plugin.QueryData) string {
 		region = "us-isob-east-1"
 	} else if strings.HasPrefix(region, "us-iso") {
 		region = "us-iso-east-1"
+	} else if d.Table.Name == "aws_s3_multi_region_access_point" {
```

Comment:
@ParthaI I don't think it's a good idea to make a specific table change in this utility function since it's used by many other tables. We'd do better to hardcode `us-west-2` in the `S3ControlMultiRegionAccessClient` function.

Also, is this available in other partitions, like US Gov, China, or US ISO?

Reply:
@cbruno10 s3 multi-region access point is available in China not available in US Gov and  US ISO.